### ISSUE TITLE
Test for WCAG failure F31: Using two different labels for the same function on different Web pages within a set of Web pages 

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#194]
http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/F31

Components that have the same function in different Web pages are more easily recognized if they are labeled consistently. If the naming is not consistent, some users may get confused. 

Note: Text alternatives that are "consistent" are not always "identical." For instance, you may have an graphical arrow at the bottom of a Web page that links to the next Web page. The text alternative may say "Go to page 4." Naturally, it would not be appropriate to repeat this exact text alternative on the next Web page. It would be more appropriate to say "Go to page 5". Although these text alternatives would not be identical, they would be consistent, and therefore would not be failures for this Success Criterion.
## Examples
- A button that says "search" in one page and to use a button that says "find" on another page.
- A button with "Save page" on one page and "Save" on another page, in both cases for the same function. 

Test procedure
1. In a set of Web pages, find components with the same function that are repeated in multiple Web pages.
2. For each component with the same function found in step #1, check that the naming is consistent.
